src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/prop-types */
+const getStyles = (btn) => {
```

Comment:
- [ ] Kindly remove the eslint disablement line and add prop-types to this button component.

Reply:
Actually, it was my editor that caused the error about importing the PropTypes from prop-types. I have fixed it now thank you so much for mentioning that. It was really helpful. 💯